public/scripts/lib/env.js

```diff
@@ -11,7 +11,8 @@ const settings = {
 	dbName: process.env.HUBOT_CLOUDANT_DB,
 	dbKey: process.env.HUBOT_CLOUDANT_KEY,
 	username: process.env.CLOUDANT_USERNAME,
-	userpass: process.env.CLOUDANT_PASSWORD
+	userpass: process.env.CLOUDANT_PASSWORD,
+	appHost: process.env.COGNITIVE_UI_HOST || 'standalone'
```

Comment:
We need consistency with the environment variables in the UI and the Bot. Some contain HUBOT_ and some don't.  

A confusing one is `HUBOT_CLOUDANT_ENDPOINT` and `HUBOT_CLOUDANT_HOST` both have a similar meaning, but looks like *_ENDPOINT expects `https://` and *_HOST expects only the hostname part.